src/main/java/recipeio/CommandValidator.java

```diff
@@ -279,14 +301,41 @@ public static boolean isValidAddCommand(String userInput) {
     public static boolean isValidFilterCommand(String userInput) {
         String[] details = InputParser.parseDetails(userInput);
         if (details.length != VALID_FILTER_LENGTH || details[INPUT_DETAILS_INDEX].isEmpty()) {
-            System.out.println("The filter function takes in one parameter: {allergy}");
-            System.out.println("\tInput Example: filter dairy");
+            System.out.println(VALID_FILTER_PROMPT);
+            System.out.println(VALID_FILTER_EXAMPLE);
             return false;
         }
         if (!isWord(details[INPUT_DETAILS_INDEX])) {
-            System.out.println("\tInput Example: filter dairy");
+            System.out.println(VALID_FILTER_EXAMPLE);
             return false;
         }
         return true;
     }
+
+    /**
+     * Checks whether the url entered is valid by checking with URLValidator
+     * Valid urls starts with 'http://', 'https://', or 'www.'
+     * Valid urls must also follow domain naming conventions
+     *
+     * @param userInput User's input in the command line.
+     * @return status of check.
+     */
+    public static boolean isValidURL(String userInput) {
+        String details = userInput.trim();
+        boolean isValid = true;
+
+        // Validate URL subdomain
+        if (!details.startsWith("http://") && !details.startsWith("https://") && !details.startsWith("www.")) {
```

Comment:
extracting the url criteria so that they are not magic literals might be a good idea!